Library/Homebrew/cask/audit.rb

```diff
@@ -540,6 +540,28 @@ def audit_livecheck_version
       false
     end
 
+    sig { void }
+    def audit_min_os
+      odebug "Auditing minimum OS version"
+
+      sparkle_min_os = audit_livecheck_min_os
+      plist_min_os = audit_plist_min_os
+      min_os_string = [sparkle_min_os, plist_min_os].max
```

Comment:
Worth noting that this will run as three audits as each audit is automatically run via looking for `audit_`-prefixed methods. It seems the intention here is to have one audit that calls these two methods, in which case you will have to rename these to something different and just have `audit_min_os` with the `audit_` prefix